packages/devtools/lib/src/service_manager.dart

```diff
@@ -384,7 +382,7 @@ class ServiceExtensionManager {
   // ignore: prefer_collection_literals
   final Set<String> _pendingServiceExtensions = Set<String>();
 
-  Completer<Null> extensionStatesUpdated = Completer();
+  var extensionStatesUpdated = Completer<void>();
```

Comment:
Why is this type `var` when the similar completer above `selectedIsolateAvailable` has type `void`?

Reply:
this was my mistake on `selectedIsolateAvailable`, it should have been `var` on the left there as well.